### DISCUSSION TITLE
iOS: Error with application password

### DISCUSSION THREAD

**Discussion by ely-see** [standardnotes/forum#2784] [General]
Hello,

in iOS I get an display error when entering the application password for opening Standard Notes. The first time I enter it there is no problem but when the app is open in the background and I'll get back, the app asks for the password but I can't enter it. For better understanding, I attach a [screenshot](https://pasteboard.co/ZBUeJZmFojlh.png)

The keyboard is at the bottom but not shown in the screenshot.

The settings are that the app asks immediately for an application password. I want to keep this setting.

Can anyone help please?

Thanks and best regards.

Andreas

**Comment by atmoio** [comment]
Hmm can't replicate this issue I'm afraid. What version of iOS, and what version of the application? Are you using the app with an account or without an account? If with an account, try uninstalling the app and reinstalling, then signing back into your account.

**Comment by ely-see** [comment]
Thanks, Mo. Tried to reinstall the app but it doesn't work either.

iOS 15.3
SN 3.11.0 (113)

Passcode immediately
Protections enabled

A lot of my notes have a protection perhaps the passcode for opening SN und viewing the protected note could be an issue?
